### DISCUSSION TITLE
Permission to execute a command for a specified number of times

### DISCUSSION THREAD

**Discussion by timyliniger** [EssentialsX/Essentials#5162] [Ideas and Feature Suggestions] [answered]
Hi all, it would be great if you could implement a new feature where you could execute commands only a certain number of times. An example:

_essentials.tp.3_ -> A user/group can execute the teleport command three times. After that the rights for this command disappear. 

A technical feature can be found for example in the plugin Plotsquared. The command "plots.claim.2" allows the user to claim two plots, for example.

**Comment by itshedwy** [comment, marked as answer]
I think, its something that you want to suggest not to Essentials but to LuckPerms. Still... possible that it'll be rejected cause you can make this via public plugins.
